src/chart/options/channel.h

```diff
@@ -118,9 +118,22 @@ struct ChannelSeriesList
 	bool removeSeries(const Data::SeriesIndex &index);
 };
 
-class Channel
+struct AxisChannelProperties
```

Comment:
Wouldn't it make sense if instead of `AxisChannelProperties`, we had an `AxisChannel` derived from `Channel`?

Reply:
The reflection system requires the creation of another class for that.